### DISCUSSION TITLE
Empty prompt textbox is expanding on various circumstances (never seen before `v0.3.35`)

### DISCUSSION THREAD

**Discussion by rotemdan** [open-webui/open-webui#6494] [Issues]
Usually happens while or after the LLM output is written:

![Screenshot_1](https://github.com/user-attachments/assets/bf7782ca-9ddd-4d3f-bbfb-ca0cf7bead33)

I've already set Rich Text Input to `off`:
![Screenshot_2](https://github.com/user-attachments/assets/064a3f9c-2249-435c-8248-4dbb369f7270)

Doesn't seem to impact it.

**Browser used**: Latest Brave (Chromium) on Windows 11.

**Comment by silentoplayz** [comment]
I think I can attest to this happening for me as well. I didn't think much of it until having came across this discussion post, which  validates the behavior I've witnessed occur occasionally.
